Jenkinsfile

```diff
@@ -290,87 +363,115 @@ pipeline {
-                    "
-                    '''
+                     # Execute the project with SOAtest CLI
+                #     ./soavirt/soatestcli \
+                #     -data ./soavirt_workspace \
+                #     -resource /parabank-jenkins/soatest/SOAtestProject/functional \
+                #     -environment 'parabank-baseline (docker)' \
+                #     -config '${soatestConfig}' \
+                #     -settings ./soavirt_workspace/parabank-jenkins/soatest/soatestcli.properties \
+                #     -report ./parabank-jenkins/soatest/report \
+                #     "
+                     '''
+                // echo '---> Parsing 9.x soatest reports'
+                // script {
+                //     step([$class: 'XUnitPublisher', 
+                //         // thresholds: [failed(
+                //         //     failureNewThreshold: '10', 
+                //         //     failureThreshold: '10',
+                //         //     unstableNewThreshold: '20', 
+                //         //     unstableThreshold: '20')
+                //         // ],
+                //         tools: [[$class: 'ParasoftSOAtest9xType', 
+                //             deleteOutputFiles: true, 
+                //             failIfNotNew: false, 
+                //             pattern: '**/soatest/report/*.xml', 
+                //             skipNoTestFiles: true, 
+                //             stopProcessingIfError: false
+                //         ]]
+                //     ])
+                // }
             }
         }
-
-        stage('Web Functional Test') {
+        stage('Selenic: Initialize Selenium Grid') {
+            when {
+                expression {
+                    return true;
+                }
+            }
             steps {
-                // Run SOAtest CLI prepped for web functional testing from docker
+                // Initialize Selenium Grid to execute Selenic tests
                 sh  '''
-                    #TODO
+                    docker run -d \
+                    --user 995:991 \
+                    --name selenium-chrome \
+                    --network demo-net \
+                    selenium/standalone-chrome:latest
                     '''
             }
         }
 
-        stage('Shift-Left Load Test') {
+        stage('Selenic: Java Selenium Test') {
+            when {
+                expression {
+                    return true;
+                }
+            }
             steps {
-                // Run Load Test CLI from docker
+                // Run Selenic from docker
                 sh  '''
-                    #TODO
+                docker run -u ${jenkins_uid}:${jenkins_gid} \
+                --rm -i --name selenic \
+                --network demo-net \
+                -v "$PWD/parabank-jenkins:/home/parasoft/jenkins/parabank-jenkins" \
+                -v "$PWD/parabank-selenic:/home/parasoft/jenkins/parabank" \
```

Comment:
The volume mounting and working directory where you run "mvn test" must change once you copy the Selenium project into this repo.